realworld/ptolemy/flattened/algebraic_heateropentank_HeaterOpenTank.elkg

```diff
@@ -0,0 +1,200 @@
+<!--
```

Comment:
Are these graphs loaded successfully? I'm not sure about comments before the opening xml tag. Indeed, I have had problems with this sort of thing in the past. :/

Reply:
They are not :D I'll change it.